src/Scene/Scene.js

```diff
@@ -75,6 +75,9 @@ define('Scene/Scene', [
 
         this.gLDebug = gLDebug;
         this.gfxEngine = c3DEngine(this,positionCamera,viewerDiv, debugMode,gLDebug);
+        this.needsUpdate = true;
+        this.forceUpdate = true;
```

Comment:
Yes, the name has changed in the latest version : https://github.com/iTowns/itowns2/pull/141/files#diff-a6b95dde1d808f9cea55c3fc36fc7f2bR190